src/Criteria/Criteria.php

```diff
@@ -26,8 +26,11 @@ public function __construct(array $filters = [], array $sorts = [], int $from =
         $this->size = $size;
     }
 
-    public function filtersToQuery(string $sql):string
+    public function filtersToQuery(string $sql): string
```

Comment:
si metoda asta si cea de `sortsToQuery` pare ca sunt folosite doar in clasa asta, nu ar trebui sa fie private?